### PR TITLE
Feature/potted plant intro

### LINKED ISSUE #729: [ FEATURE REQUEST ] - Potted plant intro event
# Overview:
As we introduced desk items, we should take more use of it! Once the player opens JN they have a chance to see Natsuki taking care of a plotted plant, preferably a cactus, the most tsundere of all plants! Natsuki could've found the plant on her teacher's desk, furthermore a cactus would be easier to take care of than other plants, which also justifies why it didn't wither before.

## Must haves:
-writing of the intro event
-The potted plant as prop

## Could haves:
- random chance to see Natsuki having the plant on her desk after entering with a unique greeting (like Nat mentions she was just caring for the plant)
- have the plant in the background, on the window ledge if she is not caring for it at the moment
- give the plant a name
- Pot of the plant could've a "Izu" logo to reference the Japanese cactus park
- mentions the cactus in the pet topic

## Is this feature request related to a problem? Please describe.
New intro!

### PR DESCRIPTION
**Pull request by Blizzardsev**
Introduces the following:

- New event `event_sanjo`; introducing a new, somewhat spiky companion for Natsuki in the classroom!
- Two new greetings:
    - `greeting_sanjo_morning`
    - `greeting_sanjo_generic`
- Desk items no longer store displayables, and take an image path (displayable created as needed when setting the item) instead to reduce memory usage
- Improved handling for desk items; supplying a `JNDeskItem` as the item parameter when calling `setDeskItem` no longer requires a slot (as this is specified on the item anyway)
- Improve `script-ch30` load sequence for greetings; greetings can now specify a desk item and expression for Natsuki to be set before the black screen fades out
- Various small adjustments/corrections

Closes #729

### CODE REVIEW

**Review by RaionArt — CHANGES_REQUESTED** [review]
The event is fun and it's nice to see Natsuki so involved and motivated!  I only noticed some small stuff that might need adjustment. 
Also didn't we plan to add a line or two to the pets topic, after the player experienced the plotted plant intro? 
Either way good job so far! I also appreciate the new facial expression while she waits for a response and like always it's nice to notice how she gets less defensive at higher affinity.

File: game/script-events.rpy [hunk 1]
```diff
@@ -2138,6 +2138,213 @@ label event_warm_package:
...
+    show natsuki option_wait_sulky
+    menu:
+        n "If you've got something you gotta ask then just say it already,{w=0.2} [player]."
+
+        "What's with the plant, [n_name]?":
+            n 1csqfl "...Seriously,{w=0.2} [player]?{w=0.75}{nw}"
+            extend 2fsqem " 'What's with the plant'?"
+            n 4fllem "And just what exactly is {i}that{/i} supposed to mean?{w=0.75}{nw}" 
+            extend 4fnmaj " Huh?"
+            n 3fcsgs "You trying to say it needs an {i}excuse{/i} to be here or something?{w=0.75}{nw}"
```
[the remark this answers is not in the document]
Comment (Blizzardsev) [reply game/script-events.rpy:2274]: Good catch; I missed this one

File: game/script-events.rpy [hunk 2]
```diff
@@ -2138,6 +2138,213 @@ label event_warm_package:
...
+            extend 3fcspo " Sheesh."
+            n 1csrbo "Talk about rolling out the red carpet."
+            n 4ccsaj "Besides."
+            n 2fslfl "{i}'The plant'{/i} has a name,{w=0.5}{nw}" 
+            extend 2ccsaj " you know."
+            n 2cllbo "..."
+            n 2cslbolsbl "..."
+            n 5csqcalsbl "...It's Sanjo."
+
+        "Looks like you and that plant both have something in common.":
```
[the remark this answers is not in the document]
Comment (Blizzardsev) [reply game/script-events.rpy:2284]: Sure, I can look at rephrasing this

File: game/script-events.rpy [hunk 3]
```diff
@@ -2138,6 +2138,213 @@ label event_warm_package:
...
+
+    play audio drawer
+    $ jnPause(4)
+
+    n "..."
+    n "...Huh?" 
+    n "Wait.{w=0.75} Is that...?"
+    n "..."
+    n "I-{w=0.2}it is!{w=0.75} What are {i}you{/i} still doing here?" 
+    n "...And what kind of inconsiderate jerk just left you {i}there{/i}?"
```
[the remark this answers is not in the document]
Comment (Blizzardsev) [reply game/script-events.rpy:2205]: Sure, I can add a little bit for this

**Review by RaionArt — APPROVED** [review]
Thank you for the adjustments! 
I love to have another wholesome intro event under out belt. Great job!

**Pull request merged**